Manifests/ManagedPreferencesApplications/com.papercut.printdeploy.plist

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_description</key>
+	<string>PaperCut Print Deploy Settings</string>
+	<key>pfm_documentation_url</key>
+	<string>https://www.papercut.com/help/manuals/print-deploy/roll-out-the-client/install-arguments/#available-parameters-and-filename-configurations</string>
+	<key>pfm_domain</key>
+	<string>com.papercut.printdeploy.client</string>
```

Comment:
I see that the full bundle ID includes a trailing _client_. Could you rename both the manifest and the icon to match?